deepchecks/vision/datasets/classification/mnist.py

```diff
@@ -198,4 +206,4 @@ def forward(self, x):
         x = F.relu(self.fc1(x))
         x = F.dropout(x, training=self.training)
         x = self.fc2(x)
-        return F.log_softmax(x)
+        return F.log_softmax(x, dim=1)
```

Comment:
How did the performance report work pre-this?

Reply:
<img src="https://c.tenor.com/bxHQ4KcM8eMAAAAC/magic-meme.gif"/>